src/API/Google/Merchant.php

```diff
@@ -314,7 +314,7 @@ public function update_account( Account $account ): Account {
 	 */
 	public function link_ads_id( int $ads_id ): bool {
 		$account   = $this->get_account();
-		$ads_links = $account->getAdsLinks();
+		$ads_links = (array) $account->getAdsLinks();
```

Comment:
I'd suggest to use the null coalescing operator to be a little more explicit that we allow a list of links or a null value. Typecasting does pretty much the same but it will also hide any other unexpected values as an array. If the function isn't returning a list of links like we expect, it should throw an error early.

```suggestion
		$ads_links = $account->getAdsLinks() ?? [];
```

Reply:
This has been addressed.